sunkit_image/radial.py

```diff
@@ -466,10 +471,17 @@ def fnrgf(
     order : `int`, optional
         Order (number) of fourier coefficients and it can not be lower than 1.
         Defaults to 3.
-    attenuation_coefficients : `float`
-        A two dimensional array of shape ``[2, order + 1]``. The first row contain attenuation
-        coefficients for mean calculations. The second row contains attenuation coefficients
-        for standard deviation calculation.
+    range_mean : `list`, optional
```

Comment:
These were renamed in the original PR and that should happen here too.

Reply:
do you suggest range_mean should be renamed to mean_attenuation_range?
and range_std to std_attenuation_range.

Reply:
Yeah, those.